gulp/tests/styles.test.js

```diff
@@ -1,8 +1,14 @@
 /* eslint-env es6 */
 /* global test, expect */
+/**
+ * External dependencies
+ */
 import { pipe as pump, from, concat } from 'mississippi';
 import Vinyl from 'vinyl';
 
+/**
+ * Internal dependencies
+ */
```

Comment:
... :)

Reply:
Hmm odd, I used ESLint with `--fix` and it generated the block style comments, not sure why it duplicated the inline style comments instead of removing them.